test/integration/retryable-writes/retryable_writes.spec.test.ts

```diff
@@ -196,3 +198,15 @@ async function turnOffFailPoint(client, name) {
     mode: 'off'
   });
 }
+
+// These tests are skipped because the driver 1) executes a ping when connecting to
+// an authenticated server and 2) command monitoring is at the connection level so
+// when the handshake fails no command started event is emitted.
+const SKIP = [
```

Comment:
same comment here about moving the manual spec test implementations from prose to this file and matching test case names